spaceship.zsh

```diff
@@ -16,19 +16,13 @@ SPACESHIP_PROMPT_TRUNC="${SPACESHIP_PROMPT_TRUNC:-3}"
 
 # PREFIXES
 SPACESHIP_PREFIX_SHOW="${SPACEHIP_PREFIX_SHOW:-true}"
-SPACESHIP_PREFIX_GIT="${SPACESHIP_PREFIX_GIT:-on}"
-SPACESHIP_PREFIX_HOST="${SPACESHIP_PREFIX_HOST:-at}"
-SPACESHIP_PREFIX_USER="${SPACESHIP_PREFIX_USER:-in}"
+SPACESHIP_PREFIX_MACHINE="${SPACESHIP_PREFIX_MACHINE:-at}"
+SPACESHIP_PREFIX_SSH_DIR="${SPACESHIP_PREFIX_SSH_DIR:-in}"
 SPACESHIP_PREFIX_ENV_DEFAULT="${SPACESHIP_PREFIX_ENV_DEFAULT:-via}"
-SPACESHIP_PREFIX_NVM="${SPACESHIP_PREFIX_NVM:-SPACESHIP_PREFIX_ENV_DEFAULT}"
-SPACESHIP_PREFIX_RUBY="${SPACESHIP_PREFIX_RUBY:-SPACESHIP_PREFIX_ENV_DEFAULT}"
-SPACESHIP_PREFIX_SWIFT="${SPACESHIP_PREFIX_SWIFT:-SPACESHIP_PREFIX_ENV_DEFAULT}"
-SPACESHIP_PREFIX_XCODE="${SPACESHIP_PREFIX_XCODE:-SPACESHIP_PREFIX_ENV_DEFAULT}"
-SPACESHIP_PREFIX_VENV="${SPACESHIP_PREFIX_VENV:-SPACESHIP_PREFIX_ENV_DEFAULT}"
-
 
 # GIT
 SPACESHIP_GIT_SHOW="${SPACESHIP_GIT_SHOW:-true}"
+SPACESHIP_GIT_PREFIX="${SPACESHIP_GIT_PREFIX:-on}"
```

Comment:
Here the same as above: put all related to `_PREFIX_` variables in corresponding section.